hack/test/day-two/ceph-values.yaml

```diff
@@ -0,0 +1,16 @@
+cephClusterSpec:
+  resources:
+    mgr:
+      limits:
+        cpu: 500m
+        memory: 512Mi
+      requests:
+        cpu: 250m
+        memory: 256Mi
+    osd:
+      limits:
+        cpu: 1000m
```

Comment:
i would just remove the limit for cpu in tests, it just throttles the app un-necessarily

Reply:
If the tests pass with the limit, I'll re-test without the limit and compare.
Thanks.